sdk/proguard-rules.pro

```diff
@@ -18,4 +18,11 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+-keep class com.cardinalcommerce.dependencies.internal.bouncycastle.**
+-keep class com.cardinalcommerce.dependencies.internal.nimbusds.**
+-keep class tech.dojo.pay.sdk.card.data.**
+-keep class tech.dojo.pay.sdk.card.entities.**
+-keep class tech.dojo.pay.sdk.payemntintent.data.**
+-keep class tech.dojo.pay.sdk.paymentMethouds.data.**
```

Comment:
since you are annotating @Keep for classes that you want to do not be obfuscated, do you need to still have them here in the proguard? or vice-versa

Reply:
yes just to avoid any problems in Future if  any new classes ben added to those packages